src/util.ts

```diff
@@ -105,7 +104,7 @@ export function checkFileType(filePath: string, fileExtToValidate: string): bool
 }
 export async function printTestDuration(vusers:string, startTime:Date, endTime : Date, testStatus : string) 
 {
-    console.log("TestRun completed\n");
+    console.log("Summary generation completed\n");
```

Comment:
Why is this changed?

Reply:
this is because i added the testRun completed log somewhere else, and this is actually printed when the summary, results and report also gets generated only.